misc/specs/atrium-specs-common/src/main/kotlin/ch/tutteli/atrium/specs/integration/CollectionAssertionsSpec.kt

```diff
@@ -12,13 +12,22 @@ import org.spekframework.spek2.style.specification.Suite
 abstract class CollectionAssertionsSpec(
     isEmpty: Fun0<Collection<Int>>,
     isNotEmpty: Fun0<Collection<Int>>,
+    sizeFeature: Feature0<Collection<Int>, Int>,
+    size: Fun1<Collection<Int>, Expect<Int>.() -> Unit>,
     describePrefix: String = "[Atrium] "
 ) : Spek({
 
     include(object : SubjectLessSpec<Collection<Int>>(
         describePrefix,
         isEmpty.forSubjectLess(),
-        isNotEmpty.forSubjectLess()
+        isNotEmpty.forSubjectLess(),
+        sizeFeature.forSubjectLess().adjustName { "$it feature" },
+        size.forSubjectLess { isGreaterThan(2) }
+    ) {})
+
+    include(object : AssertionCreatorSpec<Collection<Int>>(
+        describePrefix, listOf(1),
```

Comment:
I don't know why you have changed this from String to Int but listOf(1) and toBeDescr: 1 makes it look like there is a relation. You can either keep Int and change listOf to something like listOf(999) or go back to String

Reply:
Yes, you're right, this is indeed confusing, I'll revert it back to String.